tensorboard/plugins/scalar/summary_v2.py

```diff
@@ -30,6 +30,34 @@
 def scalar(name, data, step=None, description=None):
     """Write a scalar summary.
 
+    See also `tf.summary.image`, `tf.summary.histogram`, `tf.summary.SummaryWriter`.
```

Comment:
Similar to the other PR https://github.com/tensorflow/tensorboard/pull/4797, wdyt about either changing the text to something like "See the rest of the tf.summary.* methods for ways to write other data types", or perhaps moving these comments to the tf.summary module "overview" doc page?

Reply:
Agreed that all the tf.summary symbols are relevant, some more than others, but these are here the way they are to be conformant with the "See also" functionality in the TensorFlow devsite, lending consistency.

Reply:
Ok, caught up to the conversation on https://github.com/tensorflow/tensorboard/pull/4797#discussion_r597312820.  I guess it is a reasonably common pattern (though I'm not sure I see that it's specially handled by the tf.org software?) so in that case fine as is.